Scripts/PlayerTank.gd

```diff
@@ -1,26 +1,18 @@
 extends "res://Scripts/Tank.gd"
 
-export var rotate_left_input: String
-export var rotate_righ_input: String
-export var move_backward_input: String
-export var move_forward_input: String
-export var shoot_input: String
+export var input_code: String
+
+onready var left = input_code + "_rotate_left"
+onready var right = input_code + "_rotate_right"
+onready var backwards = input_code + "_move_backwards"
+onready var forward = input_code + "_move_forward"
+onready var shoot = input_code + "_shoot"
```

Comment:
Preferes por isto no `_ready()`?
Fiz assim para não escrever o "dobro" do código, (a declaração da variável e depois a atribuição do valor no `_ready()`)

Reply:
I agree with you, looks fine as it is 🙂